content/docs/integrations/node.md

```diff
@@ -6,38 +6,53 @@ redirectFrom:
 
 ### Introduction
 
-In this section, you will learn how to add a Postgres client to your Project and connect to your Neon Project.
+In this section, you will learn how to add a Postgres client to your NodeJS project and connect to your database.
 
-## Step 1 — Add Postgres client to your Neon project
+## Step 1 - Create a NodeJS project and add dependencies
 
-First, add a Postgres client to your Project. In this example, we used [postgres.js](https://www.npmjs.com/package/postgres), but feel free to choose another one.
+Create a NodeJS project and change to the newly created directory.
+
+```shell
+mkdir neon-nodejs-example
+cd neon-nodejs-example
+npm init -y
+```
+
+Add project dependecies using the following command:
+
+```shell
+npm install postgres dotenv
+```
 
 ## Step 2 — Store Neon credentials
 
-Next, store your Neon credentials somewhere, for example, in the `.env` file.
+Store your Neon credentials in the `.env` file. Note that for security purposes, you should not expose your Neon credentials to the browser.
 
 ```shell
-NEON_HOST=...
-NEON_DB=...
-NEON_USER=...
-NEON_PASS=...
-NEON_PORT=...
+PGHOST='<project_name>.cloud.neon.tech:<port>'
+PGDATABASE='<database>'
+PGUSER='<username>'
+PGPASSWORD='<password>'
+PROJECT_NAME='<project_name>'
 ```
 
 ## Step 3 — Connect to database using Postgres client and Neon credentials
 
-Then, to connect to the database using Postgres client and your Neon credentials, add the following code to the `pages/api/hello_worlds.js`:
+To connect to the database using the Postgres client and your Neon credentials, add the following code to the `src/app.js` file:
 
 ```javascript
-import postgres from 'postgres';
+const postgres = require('postgres');
+require('dotenv').config();
+
+const { PGHOST, PGDATABASE, PGUSER, PGPASSWORD, PROJECT_NAME } = process.env;
+const URL = `postgres://${PGUSER}:${PGPASSWORD}@${PGHOST}/${PGDATABASE}?options=project%3D${PROJECT_NAME}`;
+
+const sql = postgres(URL, { ssl: 'require' });
 
-const sql = postgres({
-  host: process.env.NEON_HOST,
-  port: process.env.NEON_PORT,
-  database: process.env.NEON_DB,
-  username: process.env.NEON_USER,
-  password: process.env.NEON_PASS,
-});
+async function getPostgresVersion() {
+  const result = await sql`select version()`;
+  console.log(result);
+}
 
-const result = await sql.unsafe(req.body);
+getPostgresVersion();
 ```
```

Comment:
we can add something like:
"run `node app.js`" and enjoy the result. You can use this in your express app, for nextjs api, etc." And add links for other integrations

Reply:
Good idea @seymourisdead . @danieltprice , what do you think?

Reply:
I think I agree with Polina's suggestion, to me this page looks a bit open-ended and missing the mention of the result of the conclusion.

Reply:
If there is an actual result you can show, that would be great as well.

Reply:
Good idea @danieltprice , I can add the result too.